db/projects.ts

```diff
@@ -313,6 +313,14 @@ export const PROJECTS = [
     repositoryURL: 'https://yast.opensuse.org',
     documentationURL: '',
   },
+  {
+    isEnable: true,
+    tag: 'infrastructure',
+    name: 'libvirt',
+    description: 'Library providing a virtualization API',
```

Comment:
```suggestion
    description: 'Libvirt provides a portable, long term stable C API for managing 
virtualization technologies',
```

Reply:
Sounds good. I pushed your suggestion minus the 'C' reference. There are numerous language bindings for the API.

Reply:
just copied the first line of the readme, contribute it back to libvirt :-)